tests/snapshots/TestCLICommands_Valid_log_file_in_flag_should_be_priortized_over_env_and_config.stdout.golden

```diff
@@ -1,7 +1,5 @@
-DEBU Found ENV var ATMOS_LOGS_FILE=/dev/stderr
+DEBU Set logs-level=debug logs-file=/dev/stdout
```

Comment:
This concerns me that something is no going to `/dev/stdout` for logging. That's fine if we're testing it's possible, but we shoudl generally NEVER send logs to `stdout` as it breaks any/all github actions that call atmos and process JSON output.

Reply:
Note, previously it was `/dev/stderr` and now it's `/dev/stdout`

Reply:
Maybe you missed the line 4 😅 
![image](https://github.com/user-attachments/assets/757b1c25-3167-4ebc-870d-571062880657)

This test checks if the code first loaded env config that had `/dev/stderr` and then overrides it with flag that had `/dev/stdout`

So the output is as per expectation.

Reply:
Agree, based on this test case, `/dev/stdout` is correct since that's the flag passed.

```
- name: "Valid log file in flag should be priortized over env and config"
    enabled: true
    snapshot: true
    description: "Test validation of env priority over config"
    workdir: "../"
    command: "atmos"
    args:
      - version
      - "--logs-level"
      - "Debug"
      - "--logs-file"
      - "/dev/stdout"
    env:
      ATMOS_VERSION_CHECK_ENABLED: "false"
      ATMOS_LOGS_FILE: "/dev/stderr"
    expect:
      diff:
        - '👽 Atmos (\d+\.\d+\.\d+|test) on [a-z]+/[a-z0-9]+'
      stdout:
        # Debug logs should be in stdout as we have choosen /dev/stdout in flag
        - "DEBU"
      exit_code: 0
```